plugins/UsersManager/lang/en.json

```diff
@@ -218,9 +218,9 @@
         "InviteActionNotes": "Please notes that resending an invite or copy invite link will extend the time limit for previous invites by %1$s days.",
         "CopyDenied": "The request is not allowed due to your browser's settings.",
         "CopyDeniedHints": "Please try again by either switching browsers or copying and sharing this link directly instead: %1$s",
-        "AuthTokenPostOnlyHelp": "Enable this option to only allow this token to be used for POST requests, this is recommended as a best security practice. The token will then not be valid as a URL parameter in GET requests.",
-        "AuthTokenPostOnlyHelpForced": "The system administrator has configured Matomo to only allow tokens to be created for use in POST requests, you cannot change this token option.",
-        "OnlyAllowPostRequests": "Only allow POST requests",
-        "PostOnly": "POST only"
+        "AuthTokenSecureOnlyHelp": "Enable this option to only allow this token to be used in a secure way (e.g. POST requests), this is recommended as a best security practice. The token will then not be valid as a URL parameter in GET requests.",
+        "AuthTokenSecureOnlyHelpForced": "The system administrator has configured Matomo to only allow tokens to be created for use in secure way (e.g. via POST requests), you cannot change this token option.",
+        "OnlyAllowSecureRequests": "Only allow secure requests",
+        "SecureOnly": "Secure only"
```

Comment:
Actually I'm wondering if we should improve that table headline. I doubt many people will understand what that should mean. And I also wouldn't know how to translate that when seeing it without any context.
Would `Secure use only` maybe be better?

Reply:
Yep, that sounds good, have updated.